test/runtests.jl

```diff
@@ -258,6 +258,16 @@ const PyInt = pyversion < v"3" ? Int : Clonglong
     end
     @test convert(BigInt, PyObject(1234)) == 1234
 
+    @static if VERSION >= v"0.7-"
```

Comment:
Maybe do this

```julia
using PyCall: hasproperty
@static if VERSION < v"0.7-"
    using PyCall: getproperty, setproperty, propertynames
end
```

in the beginning of the `runtests.jl` and run the tests even in Julia 0.6?

Reply:
I thought about it but thought that these methods will be tested by CI and julia 0.7 anyway. Doesn't hurt to add it though.